common/tokenbucket/tb.go

```diff
@@ -159,57 +160,57 @@ func (tb *tokenBucketImpl) Consume(count int, timeout time.Duration) bool {
 		}
 
 		if remTime < backoffInterval {
-			time.Sleep(time.Duration(remTime))
+			tb.clock.Sleep(remTime)
 		} else {
-			time.Sleep(time.Duration(backoffInterval))
+			tb.clock.Sleep(backoffInterval)
 		}
 
-		now := time.Now().UnixNano()
-		if now >= expiryTime {
+		now := tb.clock.Now()
+		if now.Compare(expiryTime) >= 0 {
 			return false
 		}
 
-		remTime = expiryTime - now
+		remTime = expiryTime.Sub(now)
 	}
 }
 
 func (tb *tokenBucketImpl) reset(rps int) {
 	tb.Lock()
-	tb.fillInterval = int64(time.Millisecond * 100)
+	tb.fillInterval = refillRate
```

Comment:
huh.  looks like this is never set to any other value - what do you think about simply removing it?

Reply:
makes sense, removed property.

Reply:
looks like this got lost btw.  (not really a complaint, just a FYI)

Reply:
hmm I've removed the property, so now only const refillRate is used.